Predictor_submission_template.ipynb

```diff
@@ -247,25 +267,17 @@
    "outputs": [],
    "source": [
     "# For examples, check out the examples folder:\n",
-    "# - examples/zero/Example-ZeroPredictor.ipynb\n",
-    "# - examples/linear/Example-Train-Linear-Model.ipynb\n",
-    "# - examples/lstm/Example-LSTM-Predicxtor.ipynb"
+    "# - examples/predictors/linear/Example-Train-Linear-Rollout-Model.ipynb\n",
+    "# - examples/predictors/lstm/Example-LSTM-Predictor.ipynb"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Make predictions\n",
-    "Making predictions means saving a .csv file called \"start_date_end_date.csv\" to the root folder.\n",
-    "For instance, if:\n",
-    "\n",
-    "```\n",
-    "start_date = \"2020-08-01\"\n",
-    "end_date = \"2020-08-04\"\n",
-    "```\n",
-    "\n",
-    "Then the expected output file is **2020-08-01_2020-08-04.csv**\n"
+    "Making predictions means saving a .csv file to the path specified in `output_file_path`.\n",
```

Comment:
The output file is now a param instead of assuming some name convention for the file